WalletWasabi.Fluent/Views/Settings/SettingsPageView.xaml

```diff
@@ -0,0 +1,41 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:v="using:WalletWasabi.Fluent.Views.Settings"
+             xmlns:c="using:WalletWasabi.Fluent.Controls"
+             xmlns:vm="using:WalletWasabi.Fluent.ViewModels"
+             xmlns:settings="clr-namespace:WalletWasabi.Fluent.ViewModels.Settings"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:DataType="settings:SettingsPageViewModel"
+             x:Class="WalletWasabi.Fluent.Views.Settings.SettingsPageView">
+  <UserControl.Styles>
+    <Style Selector="TabItem">
+      <Setter Property="FontSize" Value="12" />
+      <Setter Property="MinHeight" Value="50" />
+      <Setter Property="Padding" Value="0 0 24 24" />
+    </Style>
```

Comment:
Set padding to 0 and add this to styles:
```
<Style Selector="TabItem /template/ Border#PART_LayoutRoot">
  <Setter Property="Margin" Value="0 0 24 24" />
</Style>
```
This will fix the hitbox. Won't activate when the user clicks between two items or clicks under the item.